analysis/topeft_run2/run_analysis.py

```diff
@@ -273,6 +273,7 @@ def LoadJsonToSampleName(jsonFile, prefix):
             # forever until a larger worker connects.
             'resource_monitor': True,
             'resources_mode': 'auto',
+            #'filepath': f'/project01/ndcms/{os.environ["USER"]}/tmp',
```

Comment:
Please remove

Reply:
Should we consider it as a placeholder (redirecting to `/tmp` instead of redirecting to Vast) if anyone occurs in `Disk quota exceeded` issues in the local `afs` area?

Reply:
Sure, we can try `/tmp`

Reply:
done